dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -3967,6 +3967,25 @@ paths:
             type: "boolean"
       security:
       - BEARER: []
+  /users/getAllUserEmails:
```

Comment:
Don't need this, especially since the output is an array of primitives so it's definitely not needed in the frontend.